pkg/os/shell/shell_windows.go

```diff
@@ -0,0 +1,5 @@
+package shell
+
+var (
+	supportedShell = []string{"bash", "powershell"}
```

Comment:
? I guess you made a small mistake.
what about `cmd`, and instead we have `bash`?